src/main/web_src/src/components/fagsystem/tpsf/form/validation.js

```diff
@@ -480,7 +480,8 @@ export const validation = {
 			relasjoner: Yup.object({
 				partnere: ifPresent('$tpsf.relasjoner.partnere', partnere),
 				barn: ifPresent('$tpsf.relasjoner.barn', barn)
-			})
+			}),
+			identtype: ifPresent('$tpsf.identtype', requiredString).required(messages.required)
```

Comment:
Tror ikke du trenger å ha med `.required(messages.required)`. Den gjør at identtype blir required uansett hva du velger i bestillingen (been there, done that 😉  )

Reply:
Åja haha, det var den ja 😂  Da fjerner jeg 😊